spec/models/category_spec.rb

```diff
@@ -11,5 +11,8 @@
   it { is_expected.to validate_presence_of(:key) }
   it { is_expected.to validate_uniqueness_of(:key) }
   it { is_expected.to validate_presence_of(:title) }
-  it { is_expected.to validate_inclusion_of(:move_supported).in_array([true, false]) }
```

Comment:
I've just merge a pr for this 😬 so we can remove this now with the comment

Reply:
👍